extensions/graphql/src/__tests__/fixtures/graphql-test/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "graphql-test",
+  "version": "1.0.0",
+  "description": "GraphQL Test Application",
+  "keywords": [
+    "loopback-application",
+    "loopback"
+  ],
+  "main": "types/index.d.ts",
+  "engines": {
+    "node": ">=10"
+  },
+  "scripts": {
+    "build": "tsc",
+    "build:watch": "tsc --watch",
+    "clean": "rimraf dist *.tsbuildinfo .eslintcache",
+    "pretest": "npm run clean && npm run build",
+    "migrate": "node ./dist/migrate",
+    "start": "npm run build && node -r source-map-support/register .",
+    "prepare": "npm run build"
+  },
+  "repository": {
+    "type": "git"
+  },
+  "author": "",
+  "license": "MIT",
+  "files": [
+    "README.md",
+    "dist",
+    "src",
+    "!*/__tests__"
+  ],
+  "dependencies": {
+    "@loopback/boot": "^2.3.1",
+    "@loopback/context": "^3.8.1",
```

Comment:
Please remove this dependency and use `@loopback/context` instead (see your own #5625).